deps/make.blosc

```diff
@@ -89,3 +91,16 @@ $(LZ4_DIR)/lz4.o: $(LZ4_DIR)/lz4.c
 
 $(LZ4_DIR)/lz4hc.o: $(LZ4_DIR)/lz4hc.c
 	$(CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $(LZ4_DIR)/lz4hc.c
+
+$(SNAPPY_DIR)/snappy-c.o: $(SNAPPY_DIR)/snappy-c.cc
+	$(CC) $(CPPFLAGS) $(CXXFLAGS) -c -o $@ $(SNAPPY_DIR)/snappy-c.cc
```

Comment:
You have to install MinGW first, e.g. the `mingw-w64` package on Ubuntu.

You also have to fix `make.blosc` to use `CXX` as I mentioned above.

You edit the `blosc.c` source file to remove the `&& !defined(__GNUC__)` from [this line](https://github.com/Blosc/c-blosc/blob/a7695e747a6f4e9776236e2579ad90cef8ddb02a/blosc/blosc.c#L57), so that the line is simply `#if defined(_WIN32)`.

Finally, I changed to the `c-blosc-1.12.1/blosc` source directory and ran:
```
make -f ../../make.blosc CC=x86_64-w64-mingw32-gcc CXX=x86_64-w64-mingw32-g++ LIB=../../libblosc64-1.12.1.dll
```
to build the `libblosc64-1.12.1.dll` shared library.